### ISSUE TITLE
Have separate URL for latest commit on master

### ISSUE THREAD

**Issue by jpike88** [ks888/LambStatus#74]
So you and I don't have to wait to use latest commits, and this template must be based in S3, it's probably worth having another 'Create Stack' button that allows people to use a template based on the latest commit (like a 'beta' button or something).

**Comment by ks888** [comment]
Yes, I also think it's better to have a button to try out latest commits. To do that, we have at least these TODOs:
* Upload a latest cloud formation template to S3 bucket. As you said, the template has to be based in S3.
* Build & upload the binaries of lambda functions to S3 bucket. If a lambda function are modified by a commit, its binaries also have to be updated.
* Build & upload the frontend code to S3 bucket just like lambda functions.
* Let a CI service automatically do these tasks on latest commits. The repository currently uses [wercker](http://www.wercker.com/) for testing, but it seems the service doesn't have a feature to hide AWS credentials to upload things. So we need other CI service for this.

Though it's not as easy as a 'Create Stack' button, [this doc](https://github.com/ks888/LambStatus/blob/master/CONTRIBUTING.md#set-up-the-development-environment) describes how to try out latest commit for now.
